manubot/pandoc/tests/test_cite_filter/input.md

```diff
@@ -1,7 +1,7 @@
 ---
 # yaml_metadata_block with pandoc metadata
 citekey-aliases:
-  tag:meta-review: url:https://greenelab.github.io/meta-review/v/6afcab41acf01822f8af8760184cd3cb2d67ab5f/
+  meta-review: https://greenelab.github.io/meta-review/v/6afcab41acf01822f8af8760184cd3cb2d67ab5f/
```

Comment:
I added a test in https://github.com/manubot/manubot/pull/225/commits/505c7b94b71fbf6d3b6095e118fbea29c2c00cea that ensures that `tag:` and `raw:` remain unhandled. The only way back compat here would break is if we started handling these.